src/Utilities/mockoptimizer.jl

```diff
@@ -41,6 +41,7 @@ mutable struct MockOptimizer{MT<:MOI.ModelLike} <: MOI.AbstractOptimizer
     # constraints having the variable in the function. See `get_fallback`.
     eval_variable_constraint_dual::Bool
     condual::Dict{MOI.ConstraintIndex,Any}
+    conbasis::Dict{MOI.ConstraintIndex,MOI.BasisStatusCode}
```

Comment:
Nit: I understand this is consistent with the surrounding code, but the JuMP style guide recommends using underscores to separate words in a name (http://www.juliaopt.org/JuMP.jl/dev/style/#Naming-1), so `con_basis`. We'll eventually get back to this code and fix the other names, but when making additions we should follow the style guide.